addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -984,15 +983,16 @@ function makeActionManager(env) {
     }
 
     /**
-     * Helper function to find the first mobile-friendly view, if any.
+     * Helper function to find the mobile view, if any.
```

Comment:
Not really mobile anymore, could be everything, I would simply remove this line as the method's name is quite explicit